source/imt-terms.ttl

```diff
@@ -37,6 +37,11 @@
     skos:inScheme </term/imt> ;
     skos:exactMatch iana:application\/rdf\+xml .
 
+<audio/mpeg> a :EncodingFormat ;
```

Comment:
I'm thinking perhaps we should have audio/mp3 and use broadMatch to ianas mpeg? Unless we mean /imt/ should strictly mean an exactMatch. Or else we can never express exactly as to which filetype it is since MPEG could mean mp1, mp2 and mp3. Alternatively use another list all together.

Both mp3 and jpeg is in this list also: https://id.loc.gov/vocabulary/mencformat.html altough some of the members seem a bit weird I thin we could align with that one as well, for the important ones.

```
MP3 (or mp3) as a file format commonly designates files containing an elementary stream of MPEG-1 Audio or MPEG-2 Audio encoded data, without other complexities of the MP3 standard. 
[...]
The Moving Picture Experts Group (MPEG) designed MP3 as part of its MPEG-1, and later MPEG-2, standards. MPEG-1 Audio (MPEG-1 Part 3), which included MPEG-1 Audio Layer I, II and III, was approved as a committee draft for an ISO/IEC standard in 1991
```
https://en.wikipedia.org/wiki/MP3